easybuild/easyconfigs/t/Trilinos/Trilinos-12.12.1-foss-2019b-Python-3.7.4.eb

```diff
@@ -23,7 +20,7 @@ patches = [
     'Trilinos-%(version)s_py3_fixes_and_more.patch',
 ]
 checksums = [
-    '7c67d83befbeabc773661bcdfee2850c404d249160b755d3f1be2e96f564f9fd',  # trilinos-12.12.1-Source.tar.gz
```

Comment:
Please do not remove the sources and checksum of the old tarball. There might be users that already have downloaded it (like myself :slightly_smiling_face:  ) and it should still be usable.

Reply:
This PR changes from `trilinos-12.12.1-Source.tar.gz` to `trilinos_12.12.1.orig.tar.bz2`. As these are different files you'll end up with the new one downloaded and we'll never be able to consider the existing one and the old checksum.

Reply:
My idea was to keep the two checksums and do something like 
```
sources = [{
    'download_filename': ''%(namelower)s_%(version)s.new.tar.gz', 
    'filename': '%(namelower)s-%(version)s-Source.tar.gz',
}]
```
Then the existing tarballs can still be used while new tarballs can be fetched from launchpad.

However, the new tarball is compressed with `bz2` instead of `gz`, so that kills my plan. Then we can only use the new ones.